src/NServiceBus.AmazonSQS/MessagePump.cs

```diff
@@ -421,14 +421,13 @@ static bool IsMessageExpired(Message receivedMessage, Dictionary<string, string>
             }
 
             var sentDateTime = receivedMessage.GetSentDateTime(clockOffset);
```

Comment:
Is this already adjusting to local time? In the implementation for this method it is adding a clock offset to the `SentTimestamp` attribute. Is that attribute AWS Server time? If so then `sentDateTime` is already in local time and we don't need the corrected UTC, we need local UTC (which we already had).

Reply:
@mikeminutillo yes that attribute is the server time which gets adjusted with the client offset that is passed into the extension method.